### ISSUE TITLE
Hi Aaron!

### ISSUE THREAD

**Issue by Hikisti** [aaron42051/Text-Based-Dungeon-Java#1]
I forked your work for my course work. My mission is to find bugs and fix them. I also wanted to fix bugs and implement little bit Skills Class. What is need to be done in this class? I'm only doing this for my course work, not to copy your game.

Best regards,
Hikisti

**Comment by aaron42051** [comment]
Hi Hikisti,

There's a lot of bugs/inefficient coding all over this program! I'll typically point out the biggest ones at the comments at the top, the most recent version (v1.5) will say NOTED and explain the problem. The Skills class is supposed to contain a database for possible abilities the player can learn/use in combat. As of right now, there is only an Attack, Items (which currently has no function either), and Run option, and ideally there'd be a 4th called "Skills" that allow the player to use special abilities (healing, attempting to disable the enemy, etc.). The current commit is not quite updated so I'll commit again later today. Thanks for checking out my project!

Thanks again,

Aaron

**Comment by Hikisti** [comment]
Hi again Aaron!

Thanks for the quick answer! This is a nice little game. I fixed few minor bugs, can you merge my fix to the main branch, so I get full credit for my course work?

Best regards,
Hikisti

**Comment by aaron42051** [comment]
Hi Hikisti,

Do you mind explaining what you added to Skills.java?

Thanks,

Aaron

**Comment by Hikisti** [comment]
Yes, I just modified the class structure and added switch statement. I think that should be better for this than complex if..else if...else statements. I'm not really sure what do you want in this class, so I just added the switch statement and make it go trough compiler. In Shop class, I imported the ArrayList and added couple braces there.

**Comment by aaron42051** [comment]
Ok sounds good! I'll merge your fixes over.

**Comment by Hikisti** [comment]
Nice, thanks Aaron! I hope this help you to implement your Skills Class. http://examples.javacodegeeks.com/java-basics/switch-statement/java-switch-case-example/ there is instructions for using the switch statement, if you want check that out. Have nice time with implementing your game!

Best regards,
Hikisti
